src/sdk/pynni/nni/tuner.py

```diff
@@ -57,19 +57,21 @@ def generate_multiple_parameters(self, parameter_id_list, **kwargs):
 
     def receive_trial_result(self, parameter_id, parameters, value, **kwargs):
         """Invoked when a trial reports its final result. Must override.
+        By default this only reports results of algorithm-generated hyper-parameters.
+        Use `accept_customized_trials()` to receive results from user-added parameters.
         parameter_id: int
         parameters: object created by 'generate_parameters()'
         reward: object reported by trial
         """
+        # TODO: kwargs doc?
```

Comment:
remove this line?

Reply:
I'll add the doc later this week.

Reply:
But it should not appear in our repo....